pennylane_lightning/lightning_base/lightning_base.py

```diff
@@ -173,18 +181,21 @@ def execute(
             TensorLike, tuple[TensorLike], tuple[tuple[TensorLike]]: A numeric result of the computation.
         """
 
-    @abstractmethod
     def simulate(
         self,
         circuit: QuantumScript,
         state,  # Lightning [Device] StateVector
+        mcmc: dict = None,
```

Comment:
```suggestion
        *,
        mcmc: dict = None,
```

This ensure to not use mcm as positional argument. We can remove this when all the devices has support to mcm